### ISSUE TITLE
Problem with umlauts in usernames

### ISSUE THREAD

**Issue by shochdoerfer** [wslutilities/wslu#162]
**Background Information:**
 - Build: 19042.685 (Windows 10 Version 20H2)
 - WSL: Ubuntu 20.04
 - Version: 3.2.1 (installed via PPA)

**Describe the bug**
I try to create a shortcut with wslusc and it fails because my username contains an umlaut.

**To Reproduce**
My username created by Windows is `StephanHochdörfer` - In WSL2 I can commands like this ` ls -al /mnt/c/Users/StephanHochdörfer/` without any problems.

This is the wslu command I try to execute: ` wslusc -g -n MyApp /opt/myapp/bin/myapp.sh`

This is the output of wslusc:
```
[warn] wsl.ico not found in Windows directory. Copying right now...
mkdir: cannot create directory '/mnt/c/Users/StephanHochd\224rfer/wslu': No such file or directory
cp: cannot create regular file '/mnt/c/Users/StephanHochd'$'\224''rfer/wslu': No such file or directory
[info] wsl.ico copied. Located at "/mnt/c/Users/StephanHochd�rfer/wslu".
[warn] wsl-term.ico not found in Windows directory. Copying right now...
mkdir: cannot create directory '/mnt/c/Users/StephanHochd\224rfer/wslu': No such file or directory
cp: cannot create regular file '/mnt/c/Users/StephanHochd'$'\224''rfer/wslu': No such file or directory
[info] wsl-term.ico copied. Located at "/mnt/c/Users/StephanHochd�rfer/wslu".
[warn] wsl-gui.ico not found in Windows directory. Copying right now...
mkdir: cannot create directory '/mnt/c/Users/StephanHochd\224rfer/wslu': No such file or directory
cp: cannot create regular file '/mnt/c/Users/StephanHochd'$'\224''rfer/wslu': No such file or directory
[info] wsl-gui.ico copied. Located at "/mnt/c/Users/StephanHochd�rfer/wslu".
[warn] runHidden.vbs not found in Windows directory. Copying right now...
mkdir: cannot create directory '/mnt/c/Users/StephanHochd\224rfer/wslu': No such file or directory
cp: cannot create regular file '/mnt/c/Users/StephanHochd'$'\224''rfer/wslu': No such file or directory
[info] runHidden.vbs copied. Located at "/mnt/c/Users/StephanHochd�rfer/wslu".
mv: cannot move '/mnt/c/Users/STEPHA~1/AppData/Local/Temp/MyApp.lnk' to '/mnt/c/Users/StephanHochd'$'\224''rfer/OneDrive - Company/Desktop': No such file or directory
[info] Create shortcut MyApp.lnk successful
```

**Expected behavior**
The files should get created without a problem.

**Additional context**
It looks like wslvar needs to handle umlauts. When running `wslvar -s USERPROFILE` I get `C:\Users\StephanHochd�rfer` as the output.

**Comment by patrick330602** [comment]
Thank you for the bug report. I will look into the issue these days.

**Comment by patrick330602** [comment]
I have identified where the issue is. The fix is in progress now.

**Comment by patrick330602** [comment]
![Screenshot 2021-02-02 at 1 03 25 PM](https://user-images.githubusercontent.com/15316889/106554819-198a8280-6557-11eb-8b3f-37eb51a04c1b.png)

Currently fixed, will be part of 3.2.2 release
